app/controllers/api/v1/addresses_controller.rb

```diff
@@ -0,0 +1,44 @@
+module Api
+  module V1
+    class AddressesController < Api::V1::ApiController
+      def index
+        cities = current_user.addresses.includes(:city)
+
+        render json: AddressSerializer.render(cities, view: :with_associations), status: :ok
+      end
+
+      def create
+        new_address = current_user.addresses.create!(address_params)
```

Comment:
Just to check: If an exception is raised here, it will be captured at the controller layer right? (we have a `rescure_from` in `Api::V1::ApiController`)